vault/external_tests/audit/audit_filtering_test.go

```diff
@@ -289,6 +289,39 @@ func TestAuditFilteringFallbackDevice(t *testing.T) {
 	require.Equal(t, 5, numberOfEntries)
 }
 
+// TestAuditFilteringFilterForUnsupportedField validates that the audit device
+// 'filter' option fails when the filter expression selector references an
+// unsupported field and that the error prevents an audit device from created.
+func TestAuditFilteringFilterForUnsupportedField(t *testing.T) {
+	t.Parallel()
+	cluster := minimal.NewTestSoloCluster(t, nil)
+	client, err := cluster.Cores[0].Client.Clone()
```

Comment:
Why are we cloning the client here? Can't we just use the client as-is (this would also mean we don't need to set the token, either)?

Reply:
Well spotted. I think I copied the boilerplate from an existing test with multiple clients and then missed this when cleaning up the code. I've tested it now and it works and saves us time and LOC. Thanks!